tests/build.sh

```diff
@@ -88,7 +89,7 @@ nix build "$drv^first,second" --no-link --json | jq --exit-status '
     (.outputs |
       (keys | length == 2) and
       (.first | match(".*multiple-outputs-a-first")) and
-	  (.second | match(".*multiple-outputs-a-second"))))
+          (.second | match(".*multiple-outputs-a-second"))))
```

Comment:
What is going on here?

Reply:
Stray tabs converted to spaces.

Reply:
Ah I see. My bad with that, but those should be 4 spaces not 8. (Vim...)

Reply:
Fixed!